quickstart.sh

```
@@ -1984,15 +1938,21 @@ if [ "$CODEX_AVAILABLE" = true ]; then
+    hive open
 else
     echo -e "${YELLOW}Frontend build was skipped or failed.${NC} Launch manually when ready:"
-    echo -e "     ${CYAN}./hive open${NC}"
+    echo -e "     ${CYAN}hive open${NC}"
+    echo ""
+
+    echo -e "${DIM}Run ./quickstart.sh again to reconfigure.${NC}"
     echo ""
 fi
+
+fi
```

Shellcheck (0.11.0)

[error] Parsing stopped here. Is this keyword correctly matched up?

(SC1089)